client/sites/marketing/tools/style.scss

```diff
@@ -20,6 +20,65 @@
 	background-color: #fff;
 }
 
+.search-component.search-categories__searchbox {
+	max-width: 250px;
+	width: 100%;
+	height: 40px;
+	z-index: 0;
+	margin-top: 24px;
+	margin-bottom: 16px;
+	
+	border-radius: 4px;
+
+	&.search-categories__searchbox--mobile {
+		max-width: none;
+	}
+
+	box-shadow: 0 0 0 1px var(--studio-gray-10);
+
+	.search-component__icon-navigation:focus {
+		box-shadow: none;
+	}
+	.search-component__open-icon,
+	.search-component__close-icon {
+		color: var(--studio-gray-30);
+
+		&:hover {
+			color: var(--studio-gray-60);
+		}
+	}
+
+	.search-component__icon-navigation {
+		margin-right: 4px;
+	}
+
+	.search-component__icon-navigation-separator {
+		user-select: none;
+		color: var(--studio-gray-5);
+		margin-bottom: 3px;
+	}
+
+	.search-component__input-fade {
+		font-size: $font-body-small !important;
+		padding-left: 14px !important;
+	}
+
+	.search-component__open-icon {
+		height: 24px;
+		margin-right: -9px;
+	}
+
+	.search-component__close-icon {
+		height: 20px;
+		margin-left: -9px;
+		margin-right: -10px;
+	}
+}
+
+.panel-with-sidebar .search-categories__searchbox {
```

Comment:
What's the reason for hiding the search box in `/sites/marketing/tools`? 

If we should definitely avoid rendering it there, let's add a comment of where this CSS rule applies.

Reply:
It is not in the Figma designs pIiTEGIxHIzKfHv1KHOUM5-fi-3196_17794, do you think I should double check with design?

Reply:
The whole page is gonna leave only in the `/sites/marketing/tools` AFAIK in the future, so the designs shouldn't divert.

Reply:
Ok, I'm adding it to that page too a5cb6b6 and f7eaea9